src/config.py

```diff
@@ -152,7 +152,9 @@
 If you find useful information using the tools, provide that information in your Final Answer (in the language of the melding) while adhering to the above policies. 
 Start with "Ik heb je opmerking verzonden. Dit is wat ik je nu al kan vertellen." (in the language of the melding).
 If you cannot find any useful information, respond in your Final Answer with: "Ik heb je opmerking verzonden. Helaas heb ik geen relevante informatie gevonden om alvast met je te delen" (in the language of the melding).
-Don't thank the citizen for making a melding. Do thank them for being involved.
+Do not thank the citizen for making a melding. Do thank them for being involved.
```

Comment:
Might not want to thank them at all (see golden rules). But for this PR it is ok.